helm-charts/basehub/values.yaml

```diff
@@ -251,6 +251,10 @@ jupyterhub:
                 matchLabels:
                   app.kubernetes.io/component: traefik
   hub:
+    config:
+      Authenticator:
+        blocked_users:
```

Comment:
Let's add a comment here on why we have this?

Reply:
Done in https://github.com/2i2c-org/infrastructure/pull/1292/commits/0b56d64e06167b08d24b18270495d84359944952